mailbox/api/src/test/java/org/apache/james/mailbox/model/MailboxACLTest.java

```diff
@@ -30,17 +30,22 @@
 import org.apache.james.mailbox.exception.UnsupportedRightException;
 import org.apache.james.mailbox.model.MailboxACL.Entry;
 import org.apache.james.mailbox.model.MailboxACL.EntryKey;
+import org.apache.james.mailbox.model.MailboxACL.NameType;
 import org.apache.james.mailbox.model.MailboxACL.Rfc4314Rights;
+import org.assertj.core.data.MapEntry;
 import org.junit.Before;
 import org.junit.Test;
 
+import com.google.common.collect.ImmutableMap;
+
 /**
  * @author Peter Palaga
  */
 public class MailboxACLTest {
 
     private static final String USER_1 = "user1";
     private static final String USER_2 = "user2";
+    private static final boolean NEGATIVE = true;
```

Comment:
``NEGATIVE = true`` this is weird

Reply:
You're peaky. what I am supposed to do with this?

Reply:
You can let it like that, I just wanted to mention it's a bit weird but I don't see any overkill solution to it